.github/workflows/qa.yml

```
@@ -32,7 +32,7 @@ jobs:
             -   run: composer config -g github-oauth.github.com ${{ secrets.GITHUB_TOKEN }}
             -   run: composer install --prefer-dist --no-progress --no-scripts
             -   if: always()
-                run: composer run-script lint-no-warn
+                run: composer run-script lint
```

```suggestion
                run: composer run-script lint -- --format=checkstyle | cs2pr
```
En dan is deze wel nodig:

```diff
-                    tools: composer:v2
+                    tools: cs2pr
```